### DISCUSSION TITLE
How to exclude a MIME-type from a generic transcoder?

### DISCUSSION THREAD

**Discussion by udippel** [gerbera/gerbera#2985] [General]
KarlStraussberger had written deep down in some discussion, that one can include all MIME-types in the transcoding process with, like
`<transcode mimetype="audio/*" using="audio2mp3" />`
, but also exclude specific MIME-types. 
I have searched the discussions, and the most recent documentation, but didn't find any hint on how to configure the exclusion.
Like, in this example, transcode any audio, except of mp3. Which already is. 
With BuibbleUPnP I can see that any input in MP3 actually comes out in its original MP3 version, as well as transcoded MP3.

This is suboptimal, since when I hide the original, a good original MP3 still comes out as transcoded only, With the different input formats, and players, combined with `<hide-original>` and `<first-resource>`, this at times produces a mess.
Therefore, I'd rather have all non-MP3 transcoded, <hide-original>, and all MP3 just not transcoded; that is excluded in `<mimetype-profile-mappings>
`
How would I set the latter, please?

**Comment by KarlStraussberger** [comment]
https://docs.gerbera.io/en/latest/config-transcode.html#transcode

**Reply by udippel** [reply]
Thanks for the update!
